{
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
        "ID": "CVE-2016-2120",
        "ASSIGNER": "lpardo@redhat.com"
    },
    "affects": {
        "vendor": {
            "vendor_data": [
                {
                    "vendor_name": "[UNKNOWN]",
                    "product": {
                        "product_data": [
                            {
                                "product_name": "pdns",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_value": "up to and including 3.4.10"
                                        },
                                        {
                                            "version_value": "up to and including 4.0.1"
                                        }
                                    ]
                                }
                            }
                        ]
                    }
                }
            ]
        }
    },
    "problemtype": {
        "problemtype_data": [
            {
                "description": [
                    {
                        "lang": "eng",
                        "value": "CWE-190"
                    }
                ]
            }
        ]
    },
    "references": {
        "reference_data": [
            {
                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2016-2120",
                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2016-2120",
                "refsource": "CONFIRM"
            }
        ]
    },
    "description": {
        "description_data": [
            {
                "lang": "eng",
                "value": "An issue has been found in PowerDNS Authoritative Server versions up to and including 3.4.10, 4.0.1 allowing an authorized user to crash the server by inserting a specially crafted record in a zone under their control then sending a DNS query for that record. The issue is due to an integer overflow when checking if the content of the record matches the expected size, allowing an attacker to cause a read past the buffer boundary."
            }
        ]
    },
    "impact": {
        "cvss": [
            [
                {
                    "vectorString": "7.5/CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
                    "version": "3.0"
                }
            ]
        ]
    }
}
